EIPS/eip-7160.md

```diff
@@ -74,26 +109,27 @@ An open-source reference implementation of the `IERC721MultiMetadata` interface
 
 pragma solidity ^0.8.19;
 
-import "./token/ERC721/ERC721.sol";
-import "./access/Ownable.sol";
-import "./interfaces/IERC4906.sol";
-import "./IERC721MultiMetadata.sol";
+import {ERC721} from "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {IERC4906} from "@openzeppelin/contracts/interfaces/IERC4906.sol";
+import {IERC721MultiMetadata} from "./IERC721MultiMetadata.sol";
 
-contract MultiMetadata is Ownable, ERC721, IERC721MultiMetadata, IERC4906 {
-  mapping(uint256 => string[]) _tokenURIs;
-  mapping(uint256 => uint256) _pinnedURIIndices;
-  mapping(uint256 => bool) _hasPinnedTokenURI;
+contract MultiMetadata is ERC721, Ownable, IERC721MultiMetadata, IERC4906 {
+  mapping(uint256 => string[]) private _tokenURIs;
+  mapping(uint256 => uint256) private _pinnedURIIndices;
+  mapping(uint256 => bool) private _hasPinnedTokenURI;
 
-  constructor(string memory name_, string memory symbol_) ERC721(name_, symbol_) {
+  constructor(string memory _name, string memory _symbol) ERC721(_name, _symbol) Ownable() {
```

Comment:
since the constructor of Ownable doesn't require any arguments I think it could be omitted, but I'll leave it to 3rd party reviewers to decide on this minor detail